downloads/swan-lake-release-notes/swan-lake-2201.4.0/RELEASE_NOTE.md

```diff
@@ -82,6 +82,8 @@ To view bug fixes, see the [GitHub milestone for 2201.4.0 (Swan Lake)](https://g
 ## Developer tools updates
 
 ### New features
+#### OpenAPI Tool
+Added Swagger version 2 support for OpenAPI to Ballerina client, service generation.
```

Comment:
```suggestion
Added Swagger version 2 support for the OpenAPI to Ballerina client and service generations.
```

Reply:
@lnash94 @praneesha @aneeshafedo Shall we simply say `Added support to generate Ballerina client and service declarations from OpenAPI 2.0(i.e. swagger) definitions`? This will convey the idea more clearer IMO.

Reply:
How about using `Swagger 2.0` instead of `OpenAPI 2.0` since the term "Swagger 2.0" is more commonly used to refer to this version of the specification;
`Added support to generate Ballerina client and service declarations from Swagger 2.0(i.e. OpenAPI 2.0) definitions`